### DISCUSSION TITLE
Adobe Color Profile Pulldown

### DISCUSSION THREAD

**Discussion by Dehazeme** [rsjaffe/MIDI2LR#577] [Q&A] [answered]
Is there a way to assign the profiles contained in the Adobe color profile pulldown (color, landscape, portrait, etc.)? Thanks in advance.

**Comment by rsjaffe** [comment, marked as answer]
No. Adobe changed how that works, and it is undocumented and unsupported by their API. I've put in a request for an API but that was ignored.

**Comment by Dehazeme** [comment]
Thank you so much.

 

Being a newbie, I hate to pollute discussion forums with basic questions. I have encountered what seems to be a bug in the MIDI2LR handling of the new LR masking tools.  Should I post the question, or is it better to just ask you the expert via Email?

 

Thanks,

 

Jack

**Reply by rsjaffe** [reply]
post as a new problem. I'll just tag it closed if it's not a defect, or convert it into an issue for the next release if it is.
